packages/server/modules/cli/commands/db/helpers/index.ts

```diff
@@ -0,0 +1,19 @@
+import { getAllRegisteredDbClients } from '@/modules/multiregion/dbSelector'
+
+export type CommonDbArgs = {
+  regionKey?: string
+}
+
+export const getTargettedDbClients = async (params: { regionKey?: string }) => {
```

Comment:
isn't this the same function as [this](https://github.com/specklesystems/speckle-server/blob/6d753fe3fd437f942a0c1783ff6a93ddad668356/packages/server/modules/multiregion/dbSelector.ts#L56) ?

Reply:
getDb - get specific DB
getTargettedDbClients - get one or many DBs depending on the --regionKey CLI arg. if its undefined, this will return all region DBs plus the main one.

this is common logic that would have to be replicated over and over again in each CLI command otherwise